test/karma/karma.conf.js

```diff
@@ -0,0 +1,114 @@
+// Karma configuration
+// http://karma-runner.github.io/0.12/config/configuration-file.html
+// Generated on 2014-10-07 using
+// generator-karma 0.8.3
+
+module.exports = function(config) {
+  'use strict';
+
+  config.set({
+    // enable / disable watching file and executing tests whenever any file changes
+    autoWatch: true,
+
+    // base path, that will be used to resolve files and exclude
+    basePath: '../../',
+
+    // testing framework to use (jasmine/mocha/qunit/...)
+    frameworks: ['jasmine'],
+
+    // list of files / patterns to load in the browser
+    files: [
+    
+    //global mocks
+    'test/karma/mocks/test-mocks.js',
+
+    "app/bower_components/jquery/dist/jquery.js",
+    "app/bower_components/angular/angular.js",
+    "app/bower_components/angular-cookie/angular-cookie.js",
+    "app/bower_components/angular-facebook/lib/angular-facebook.js",
+    "app/bower_components/angular-route/angular-route.js",
+    "app/bower_components/angular-ui-router/release/angular-ui-router.js",
+    "app/bower_components/ng-debounce/angular-debounce.js",
+    "app/bower_components/angular-password-strength/build/angular-password-strength.min.js",
+    "app/bower_components/autofill-event/src/autofill-event.js",
+    //"app/bower_components/crypto-js/components/core.js",
+    //"app/bower_components/crypto-js/components/ripemd160.js",
+    "app/bower_components/moment/moment.js",
```

Comment:
I'm in China and have no VPN right now, so I can't install anything off googlecode. Will look into adding cryptojs later...